src/components/TodoList.tsx

```diff
@@ -67,7 +69,7 @@ const TodoList = () => {
           center
           icon={
             <BiTag
-              style={{ transform: 'rotate(180deg)' }}
+              className={classes.todoListArrow}
               color="gray"
```

Comment:
I'd move the color to the class, and you could look into the colors provided by Mantine https://mantine.dev/theming/extend-theme/